### PR TITLE
Raise value error for min max computation of feature range

### PR DESCRIPTION
**Pull request by kicha0**
Raising value error for exception in converting min / max of a feature column to float, and printing the type of the argument.

## Description

Observed value error:
<class 'ValueError'>
File "/mnt/azureml/cr/j/[guid]/exe/wd/_telemetry/_loggerfactory.py", line 143 in wrapper
return func(*args, **kwargs)
File "create_rai_insights.py", line 184 in main
_ = RAIInsights(
File "/azureml-envs/responsibleai/lib/python3.8/site-packages/responsibleai/rai_insights/rai_insights.py", line 236 in __init__
self._feature_ranges = RAIInsights._get_feature_ranges(
File "/azureml-envs/responsibleai/lib/python3.8/site-packages/responsibleai/rai_insights/rai_insights.py", line 1263 in _get_feature_ranges
max_value = float(test[col].max())

## Checklist

- [x] I have added screenshots above for all UI changes.
- [x] I have added e2e tests for all UI changes.
- [x] Documentation was updated if it was needed.

### CODE REVIEW

**Review by gaugup — CHANGES_REQUESTED** [review]
Could you please add a unit test for this change?

**Comment by kicha0** [comment]
> Could you please add a unit test for this change?

Hi, In the process of trying to troubleshoot we find missing exception information about the failure, this is exception handling (logging) for a case with unknown trigger condition.  Could you suggest a suitable unit test for this as it's not clear what a dataset needs to look like in the first place to trigger a ValueError in this part.

File: responsibleai/responsibleai/rai_insights/rai_insights.py [hunk 1]
```diff
@@ -1279,8 +1279,18 @@ def _get_feature_ranges(
...
             else:
-                min_value = float(test[col].min())
-                max_value = float(test[col].max())
+                col_min = test[col].min()
+                col_max = test[col].max()
+                try:
+                    min_value = float(col_min)
+                    max_value = float(col_max)
+                except Exception as e:
+                    raise ValueError(
```
Comment (gaugup) [inline responsibleai/responsibleai/rai_insights/rai_insights.py:1288]: Could you change this to SystemErrorException?

Comment (kicha0) [reply responsibleai/responsibleai/rai_insights/rai_insights.py]: done.

**Review by gaugup — APPROVED** [review]

**Pull request merged**